hedera-mirror-rest/__tests__/specs/accounts/{id}/rewards/all-params.json

```
@@ -0,0 +1,52 @@
+{
+  "description": "Account staking reward payouts api call with all applicable params",
+  "setup": {
+    "entities": [
+      {
+        "num": 1001
+      },
+      {
+        "num": 800
+      }
+    ],
+    "stakingRewardTransfers": [
+      {
+        "accountId": "0.0.1001",
+        "amount": 10,
+        "consensus_timestamp": "123456789000011111",
+        "payerAccountId": "0.0.800"
```

```suggestion
        "payer_account_id": "0.0.800"
```

In addition, I don't think payer account id matters in any of your test cases, so can just remove it

---

Done.